upgrade_tests/test_scenarios.py

```diff
@@ -286,3 +286,246 @@ def test_post_user_scenario_bug_1429201(self):
         self.assertIsNotNone(container_ids)
         self.assertIn('Error', result_fail.values()[0])
         self.assertIn('Complete', result_pass.values()[0])
+
+
+class Scenario_capsule_sync(TestCase):
+    """The test class contains pre-upgrade and post-upgrade scenarios to test if
+    package added to satellite preupgrade is synced to capsule post upgrade.
+
+    Test Steps:
```

Comment:
We do use that format for test methods and not for Test Class. Also If I run ```make dos clean``` then it doesnt shows any problem as the travis is passed.